lib/internal/mime.js

```diff
@@ -0,0 +1,279 @@
+'use strict';
+const {
```

Comment:
most of our files have a blank line after use strict

Reply:
fixing